core/domain/Domain.php

```diff
@@ -81,4 +88,14 @@ public static function brandName()
     {
         return (string) apply_filters('FHEE__EventEspresso_core_domain_Domain__brandName', 'Event Espresso');
     }
+
+
+    /**
+     * @return bool
+     * @since $VID:$
+     */
+    public function isMultiSite(): bool
```

Comment:
Return type declaration only works in PHP 7+. Is that fine?

Reply:
yes, this is based off of the dev branch which bumps the min PHP version requirement to 7.1.
great observation, thnx.